### ISSUE TITLE
b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#129]
**Description:**

The purpose of this run is to test the salt flux limiter implemented in CICE6 (based on CICE5 salt flux limiter).

The run will use the models: MOM6 + CICE6 + cam_dev.

We moved to a new sandbox that will include Adam's branch (https://github.com/adamrher/CAM.git, branch cam6_3_058.dtsens) for the atmosphere model instead of cam6_3_041.

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007

On github:
https://github.com/NCAR/amwg_dev/tree/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM

On github:
N/A

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007/atm/](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007/atm/)

**Contacts:**
@cecilehannay and @gustavo-marques

**Comment by jedwards4b** [comment]
The github location of this sandbox is https://github.com/ESCOMP/CESM branch cesm3_coupled_test
commit 73d5bb35d9

**Comment by justin-richling** [comment]
ADF diagnostics have been uploaded [here](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007/atm/ADF/b.cesm3_cam058_mom.B1850WscMOM.ne30_L58_t061.007_1_10_vs_b.e21.B1850.f09_g17.CMIP6-piControl.001_1_10/).

**Comment by gustavo-marques** [comment]
The simulation completed 20 years. Salt is conserved within the ocean model. As in run 008 (https://github.com/NCAR/amwg_dev/issues/130), convection in the Lab Sea shuts off after ~ 6 years (see plot below). 
![image](https://user-images.githubusercontent.com/11339137/173670301-498d54b4-b3aa-4783-aaa5-fe823c67e320.png)

In terms of salt conservation in the ice model, I noticed that the "arwt salt flx error" is always (I only looked at the last log file) much larger in the Arctic when compared to the Antarctic (see an example below). Perhaps this is because sea ice is accumulating in the Arctic. @dabail10, any thoughts? 

                           Arctic                 Antarctic
arwt salt flx error    =   -4.34887567972258312E-07   2.77767027356864325E-17

**Comment by dabail10** [comment]
This is concerning. I wonder if it is related to the 4psu "fix". I haven't seen this difference in the errors previously. The Lab Sea freezing over is another issue which is going to be trickier.

**Comment by adamrher** [comment]
What has RESTOM looked like in all these b-case runs using this new code base [cesm3_cam6_3_058_MOM](https://github.com/NCAR/amwg_dev/tree/cesm3_cam6_3_058_MOM)? I just found out that the atmosphere is colder in this code base, compared to earlier ones. I found out why, and can fix it so it recovers a climate we have been using. Don't get your hopes up that this will stop the Lab sea from freezing, but it's moving in the right direction. Thoughts?

**Comment by gustavo-marques** [comment]
My hopes that making the atmosphere warmer will stop the Lab sea from freezing are very high.

**Comment by cecilehannay** [comment]
@adamrher: It looks like RESTOM is missing form the ADF diags. I will post the AMWG diags so we can look at RESTOM.

**Comment by cecilehannay** [comment]
But from the corresponding F cases (#112 and #132), I expect the RESTOM to be lower. It is 1 W/m2 colder in the B case.

**Comment by JulioTBacmeister** [comment]
The numbers I posted are from some IDL scripts I have run for 10 years from the h0 files.  The numbers from my scripts agreed in the past with those from AMWG. FLNT and FSNT agree with ADF for 007.  I think the global numbers for FSNT and FLNT should be written with more digits in ADF - they are currently format NNN maybe should be NNN.nn

**Comment by adamrher** [comment]
Julio showed [here](https://github.com/NCAR/amwg_dev/issues/132#issuecomment-1156912581) that this [cesm3_cam6_3_058_MOM](https://github.com/NCAR/amwg_dev/tree/cesm3_cam6_3_058_MOM) code base seems to result in about a 0.5 W/m2 reduction in RESTOM, compared to the code we were using before.

I have updated my CAM branch to restore this previously warmer climate. @jedwards4b @cecilehannay the new CAM hash to add to your Externals.cfg is [7e6a845](https://github.com/adamrher/CAM/commit/7e6a845779cf72fc01399d8f602fcbf4ef262b1c).

**Comment by jedwards4b** [comment]
I have updated https://github.com/ESCOMP/CESM/pull/new/cesm3_cam6_3_058_MOM with the new cam tag.

**Comment by cecilehannay** [comment]
I can do a F case with the new tag.

**Comment by adamrher** [comment]
Should we do a B case? To see impacts on Lab sea? Or do we want to understand the salt issues a bit more first?

**Comment by cecilehannay** [comment]
I was suggesting a F case to make sure the climate is the same as in cam6_3_041 but maybe you check this already.

**Comment by JulioTBacmeister** [comment]
Adam could extend his F-case with revice, while Cecile starts a B-case.

**Comment by cecilehannay** [comment]
Yes, but one thing is that I was going to do the F case with the same sandbox as the one we will use for the B case. I know in theory, cam should be the same in Adam's sandbox and mine, but... 

I am going to meet with Gustavo to talk about the B case.

**Comment by adamrher** [comment]
I'd say lets forgo the F case until we want to do another F case experiment, at which point we can redo the F case control.

Maybe just do a new B case, and check after 10 years whether the climate is similar to the older B case runs with the older sandbox.

**Comment by cecilehannay** [comment]
I just talked with @gustavo-marques and the salinity issues are not solved yet. They need to look at this more.

**Comment by JulioTBacmeister** [comment]
Agree. If the new B case is ready to go let’s start it.

I say use the old topography for this.

Then let’s swap topo if it looks ok

**Comment by gustavo-marques** [comment]
I suggest running an F case using the updated sandbox (cesm3_cam6_3_058_MOM) to make sure we are getting the same climate as before. In the meantime, I will look into the salinity issue in CICE with @dabail10. We need to understand and solve this issue before running a new B case.

**Comment by JulioTBacmeister** [comment]
Is this an issue with CICE6?

If it is I’d consider a run with Adam’s cam058+revice+cice5

**Comment by gustavo-marques** [comment]
It's an issue with both CICE5 and CICE6. Both versions use the hack that avoids CICE from taking more salt that's available in the ocean model.
